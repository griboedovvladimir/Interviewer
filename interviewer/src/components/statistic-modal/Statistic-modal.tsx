import * as React from "react";
import {Component} from 'react';

export class StatisticModal extends Component {
    public props:any;
    public state:any;
    constructor(props:any){
        super(props);
    }

    public render() {
        return (
            <div>
                <div className="remove">
                    <i id="remove-modal" className="material-icons">close</i>
                </div>
                <div>{this.props.particularQuestion[0].text}</div>
            </div>
        )
    }
}

