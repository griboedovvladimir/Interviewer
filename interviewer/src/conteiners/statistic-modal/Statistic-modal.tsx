import * as React from "react";
import {Component} from 'react';

export class StatisticModal extends Component {
    constructor(props:any){
        super(props)
    }

    public render(){
        return (
            <h1></h1>
        )

}