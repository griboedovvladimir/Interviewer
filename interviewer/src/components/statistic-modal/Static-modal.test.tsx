import {shallow} from "enzyme";
import {StatisticModal} from "./Statistic-modal";
import * as React from "react";
import "isomorphic-fetch";

const questionObj = {
    question_id: '15',
    subtopic_id: '2',
    ratting: '7',
    text: 'text',
    source: 'text'
};

let wrapper:any;
describe('>>>StatisticModal component', () => {
    beforeEach(() => {
        wrapper = shallow(<StatisticModal particularQuestion = {[questionObj]} />)
    });

    it('render simple StatisticModal component', () => {
        expect(wrapper.find('div.remove').length);
        expect(wrapper).toMatchSnapshot();
    });

});