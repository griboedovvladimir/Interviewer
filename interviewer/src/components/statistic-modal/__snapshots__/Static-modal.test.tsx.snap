// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>StatisticModal component render simple StatisticModal component 1`] = `
<div>
  <div
    className="remove"
  >
    <i
      className="material-icons"
      id="remove-modal"
    >
      close
    </i>
  </div>
  <div>
    text
  </div>
</div>
`;
